public/js/components/Accordion.js

```diff
@@ -15,7 +15,7 @@ const Accordion = React.createClass({
 
   getInitialState: function() {
     return { opened: this.props.items.map(item => item.opened),
-             created: [] };
+      created: [] };
```

Comment:
a lot of these formatting issues came from an early formatting decision to keep the first key on the the `{` line.


At this point, when an object is longer than 80 chars, I believe we're consistently using this format:

```js
foo({
  bar: 'baz',
  foo: 'bar'
})
```

Many of these changes still need to be updated to meet this format. Would you be able to do that?